eng/pipelines/runtimelab/install-llvm.ps1

```diff
@@ -90,8 +107,12 @@ foreach ($Config in $Configs | % { if ($_ -eq "Checked") { "Debug" } else { $_ }
         }
     }
     $CmakeConfigureCommandLine += "-DCMAKE_BUILD_TYPE=$LlvmConfig"
+    
+    if (!$IsWindows)
+    {
+        $CmakeConfigureCommandLine += "-DCMAKE_SYSROOT=/crossrootfs/x64", "-DCMAKE_SYSTEM_NAME=Linux", "-DCMAKE_INSTALL_PREFIX=/usr/local/llvm-cross", "-DLLVM_HOST_TRIPLE=x86_64-pc-linux-gnu", "-DLLVM_TARGETS_TO_BUILD=WebAssembly"
```

Comment:
Why is `DLLVM_HOST_TRIPLE`/`DCMAKE_SYSTEM_NAME` needed? Same question for `DLLVM_TARGETS_TO_BUILD` (we don't build target-specific libraries)?

Reply:
I think we can drop `DLLVM_HOST_TRIPLE` and `DLLVM_TARGETS_TO_BUILD `.  I was following https://llvm.org/docs/HowToCrossCompileLLVM.html but this is not full cross compiling, so guess we don't need them all.  Attempting to drop `DCMAKE_SYSTEM_NAME ` fails:

```
../../../bin/llvm-min-tblgen: error while loading shared libraries: libtinfo.so.5: cannot open shared object file: No such file or directory
include/llvm/TargetParser/CMakeFiles/RISCVTargetParserTableGen.dir/build.make:118: recipe for target 'include/llvm/TargetParser/RISCVTargetParserDef.inc' failed
```